README.md

```diff
@@ -55,383 +56,249 @@ of the [Federated Identity Community Group](https://fedidcg.github.io/).
+const token = (await fetch("https://example.com/login?rp=rp.example&id=" + id)).text();
+// Do whatever you would usually do with your token here.
+```
 
-## Key scenarios
+This approach works well with the Passive mode: if the user is not already signed in to the IdP, the browser doesn’t need to display anything. In Active Mode, the RP may wish to provide some guidance or trigger some action to get the user to refresh their IdP session, like navigating to the login page if it is known.
 
-These APIs together enable login and linking scenarios that I have put into a few categories.
-For all of these, imagine that an identity provider would store a credential on the user's browser when they log in.
+> [!NOTE]
+> We refer to a `url` field on the RP-side provider configurations instead of `configURL`; this is to align with future work deprecating the 
+> `configURL` parameter in favor of `url` in FedCM.
 
+### Active Mode: Handling the Signed-Out Case
 
-### Scenario 1: User intends to link to an identity provider they are not logged into
+For Active mode, if you want to be able to gracefully handle the signed-out case, IdPs have an incremental path: allow users to login using the “url” passed by the RP or (optionally) the IdP providing a custom one.
 
-In this case, our user is not even logged into this this identity provider (`idp.net`), just having navigated to this site (`example.com`).
-They first interact with some UI in the page that is clearly associated with the identity provider and the following is called.
+So, if an RP called:
 
-```js
-let credential = await navigator.credentials.get({
-  identity : {
-    providers : [
-      {
-        loginURL : "https://login.idp.net/login.html",
-        origin: "https://login.idp.net", // may be omitted, inferred from loginURL
-      },
-    ]
-  }
+```
+let cred = await navigator.credentials.get({
+	identity: {
+         mode: "active",
+         providers: [{url: "https://example.com"}]
+       },
 });
 ```
 
-The browser sees there is no credential in the credential store that would work for `example.com`.
-So it falls back and opens the `loginURL`. The RP can choose whether to open this URL in a pop-up or via a redirect.
-The API defaults to redirecting the user to the `loginURL`.
-There, the user goes through some authentication and/or authorization flow entirely of the identity provider's choosing, after which the identity provider stores a credential with some code like this:
+The browser would check the sign-in status list, 
+first by trying to resolve “https://example.com/.well-known/web-identity”. If the well-known file doesn’t exist (i.e., the request returns a status 404), the browser would use `“login_url”` as the value of `“https://” + eltd+1(“url”)` (i.e., in this example, `“https://example.com”`). If an IdP requires a custom `login_url`, they can have a very simple `.well-known/identity` file, such as the following:
```

Comment:
A further alternative could be to allow navigation to the full URL if the user accepts an IDP-dialog if there is some indication the url is actually a participating IDP (like prior registration or a origin-level well-known)